modules/microshift-disconnected-host-procedure.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-disconnected-network-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disconnected-host-network-config_{context}"]
+= Configuring the networking settings for fully disconnected hosts
+
+To configure the networking settings for running {microshift-short} on a fully disconnected host, you must prepare the host, update the networking configuration, then restart to apply the new settings. All commands are executed from the host CLI.
+
+.Prerequisites
+* RHEL 9 or newer.
+* {microshift-short} 4.14 or newer.
+* Access to the host CLI.
+* A valid IP address chosen to avoid both internal and potential future external IP conflicts when running {microshift-short}.
+* {microshift-short} networking settings are set to defaults.
+
+[IMPORTANT]
+====
+The following procedure is for use cases in which access to the {microshift-short} cluster is not required after devices are deployed in the field. There is no remote cluster access after the network connection is removed.
+====
+
+.Procedure
+
+. Add a fake IP address to the loopback interface by running the following command:
++
+[source,terminal]
+----
+$ IP="10.44.0.1" <1>
+$ sudo nmcli con add type loopback con-name stable-microshift ifname lo ip4 ${IP}/32
+----
+<1> The fake IP address used in this example is “10.44.0.1”.
++
+[NOTE]
+====
+Any valid IP works, as long as it avoids both internal {microshift-short} and potential future external IP conflicts. This can be any subnet that does not collide with the {microshift-short} node subnet or is be accessed by other services on the device.
```

Comment:
```suggestion
Any valid IP works, if it avoids both internal {microshift-short} and potential future external IP conflicts. This can be any subnet that does not collide with the {microshift-short} node subnet or is be accessed by other services on the device.
```

Reply:
Personally, I LIKE "as long as", but lordy the style guide doesn't. https://www.ibm.com/docs/en/ibm-style?topic=word-usage#as-long-as